i18n/fr/docusaurus-plugin-content-docs/version-23.04/integrations/data-analytics/sc-splunk-events.md

```diff
@@ -46,68 +49,88 @@ apt install centreon-stream-connector-splunk
+| Type   | Nom             | Explication                | Exemple de valeur                                           |
 | ------ | --------------- | --------------------------------------- | ------------------------------------------------------- |
-| string | http_server_url | the url of the Splunk service collector | `https://mysplunk.centreon.com:8088/services/collector` |
-| string | splunk_token    | Token to use the event collector api    |                                                         |
+| string | http_server_url | L'URL du collecteur de services de Splunk | `https://mysplunk.centreon.com:8088/services/collector` |
+| string | splunk_token    | Jeton pour l'API du collecteur d'évènements    |                                                         |
+
+6. Renseignez les paramètres optionnels désirés (en utilisant le lien **+Add a new entry**) :
+
+| Type   | Nom               | Explication                                             | Valeur par défaut                              |
+| ------ | ----------------- | --------------------------------------------------------------- | ------------------------------------------- |
+| string | logfile           | Fichier dans lequel les logs sont écrits                              | /var/log/centreon-broker/splunk-events.log |
+| number | log_level         | Niveau de log, de 1 (erreurs) à 3 (débug)                      | 1                                           |
 
-### Add Splunk optional parameters
+7. Utilisez les paramètres optionnels du stream connector pour [filtrer ou adapter les données que vous voulez que Centreon envoie à Splunk](#filtrer-ou-adapter-les-données-que-vous-voulez-envoyer-à-splunk).
 
-Some stream connectors have a set of optional parameters dedicated to the Software that they are associated with. To add them you must **click** on the **+Add a new entry** button located **below** the **filter category** input.
+8. [Déployez la configuration](../../monitoring/monitoring-servers/deploying-a-configuration.md).
 
-| Type   | Name              | Value explanation                                               | default value                              |
-| ------ | ----------------- | --------------------------------------------------------------- | ------------------------------------------ |
-| string | splunk_sourcetype | Identifies the data structure of the event                      | _json                                      |
-| string | splunk_host       | Name or address of the server that generated the event          | Central                                    |
-| string | splunk_index      | Index where the events are stored                               |                                            |
-| string | splunk_source     | source of the http event collector. like `http:<name_of_index>` |                                            |
```

Comment:
we can't just remove this information without telling people that they do exist somewhere and that they have a default value

Those parameters are dedicated to the splunk stream connector therefore, they aren't liste in the github documentation with all documentation.